plugin.py

```diff
@@ -19,7 +19,6 @@ class LspCspellPlugin(NpmClientHandler):
     package_name = __package__
     server_directory = 'language-server'
     server_binary_path = os.path.join(server_directory, '_server/dist', 'main.js')
```

Comment:
Not likely to cause any issues but should be:

```suggestion
    server_binary_path = os.path.join(server_directory, '_server', 'dist', 'main.js')
```